src/components/mobile/mobile-aside.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div
+    v-if="$x.totalResults > 0"
+    class="
+      x-list x-list--vertical x-list__item--expand
+      x-padding--05
+      x-list--padding-left x-list--padding-right
+    "
+  >
+    <BaseScroll class="x-list__item--expand">
+      <Sort class="x-list--padding-04 x-list--padding-bottom" />
```

Comment:
These classes can be deleted if the facets headers spacings are set using the padding token.
Also the CustomFacets ones.